jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-backend/src/main/java/org/jbpm/workbench/cm/backend/server/RemoteCaseManagementServiceImpl.java

```diff
@@ -46,6 +54,14 @@
 public class RemoteCaseManagementServiceImpl implements CaseManagementService {
 
     public static final int PAGE_SIZE_UNLIMITED = Integer.MAX_VALUE;
+    public static List<String> allCaseActionStatus = new ArrayList<String>() {
```

Comment:
This actually create an inner class for the list, my suggestion would be to use `Arrays.asList` to build this list.